Commands.adoc

```diff
@@ -0,0 +1,244 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you `vagrant up` and have a running VM, the vagrant-service-manager
+ enables you to check and tweak your container development environment.
+
+'''
+toc::[]
+'''
+
+You can use the vagrant-service-manager to set up your environment variables and
+ get the TLS certificates to secure the Docker communication channel; identify
+ the routable ip address and version of your VM; and check the status and
+ manage the lifecycle of the configured services in the running VM.
+
+To run the vagrant-service-manager plugin use: +
+-------------------------------------------
+vagrant service-manager [command] [options]
+-------------------------------------------
+The following commands can be used: +
+
+`env`::  Displays the connection information for services in the VM +
+`box`:: Displays the VM related information like version, release, IP etc +
+`status`:: Lists services and their running state +
+`start`:: Starts the given service in the VM +
+`stop`:: Stops the given service in the VM +
+`restart`:: Restarts the given service in the VM +
+
+The following options can be used: +
+
+`--script-readable`:: Displays information in a key=value format +
+`-h, --help`:: Prints help for the specific command being run +
+
+
+== Get Environment Variables and Certificates
+
+If you need to get the environment variables and certificates for all the active
+ services in the VM, run: +
+
+`$ vagrant service-manager env` +
+
+Usage: `vagrant service-manager env [service] [options]`
+
+The possible options for service are `docker` and `openshift`. +
+If no service is specified `vagrant service-manager env` provides connection
+ information for all the active services.
+
+The following options can be used: +
+
+`--script-readable`:: Displays information in a key=value format +
+`-h, --help`:: Prints help for the specific command being run. +
+
+=== Examples
+
+If you need connection information for all active services in the VM, in a
+manner that can be evaluated in a shell, use:
+-----------------------------------------------------------------------------
+$ vagrant service-manager env
+
+# docker env:
+# Set the following environment variables to enable access to the
+# docker daemon running inside of the vagrant virtual machine:
+export DOCKER_HOST=tcp://10.1.2.2:2376
+export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+export DOCKER_TLS_VERIFY=1
+export DOCKER_API_VERSION=1.21
+
+# openshift env:
+# You can access the OpenShift console on: https://10.1.2.2:8443/console
+# To use OpenShift CLI, run: oc login https://10.1.2.2:8443
+export OPENSHIFT_URL=https://10.1.2.2:8443
+export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
+export DOCKER_REGISTRY=hub.openshift.centos7-adb.10.1.2.2.xip.io
+
+# run following command to configure your shell:
+# eval "$(vagrant service-manager env)"
+-----------------------------------------------------------------------------
+
```

Comment:
True, it should, but it doesn't atm. So I'd rather be quiet for now. This is an outstanding issue we have which we need to address as part of the efforts to support all possible Windows terminals.